pkg/common/cns-lib/volume/manager.go

```diff
@@ -515,6 +557,70 @@ func (m *defaultManager) createVolumeWithImprovedIdempotency(ctx context.Context
 	return resp, faultType, err
 }
 
+func (m *defaultManager) waitOnTask(ctx context.Context, taskMoRef vim25types.ManagedObjectReference) (*vim25types.TaskInfo, error) {
+	listViewCtx := context.TODO()
```

Comment:
why do use "context.TODO()" and not use the supplied `ctx`?

Reply:
Fixed.